debezium-connector-mongodb/pom.xml

```diff
@@ -34,6 +34,21 @@
             <groupId>org.mongodb</groupId>
             <artifactId>mongodb-driver-sync</artifactId>
         </dependency>
+        <dependency><!--// For Debezium MongoDB Connect REST Extension, @see DebeziumMongoDbConnectRestExtension //-->
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>connect-runtime</artifactId>
+            <scope>provided</scope>
```

Comment:
@Naros isn't this what you asked for? Provided scope is similar to compile-time deps, isn't it?

Reply:
AFAIK provieded is more similar to runtime actually with the exception of not pulling down the dependency by the current maven reactor during the build but rather it is deferred to the user/runtime platform to provide the dependency during runtime

Reply:
Ah yes, I think it is required as we need access the Kafka Connect `Herder` and consume some Kafka Connect datastructures that are in the `connect-runtime` package.

Reply:
I have removed this dependency from the connector modules in a kind of elegant way, I assume. :crossed_fingers: Thx for pushing me, after some time I found a hopefully good solution.